cfme/automate/service_dialogs.py

```diff
@@ -116,6 +116,19 @@ def __init__(self, label=None, description=None,
         self.box_label = box_label
         self.box_desc = box_desc
 
+    def add_element(self, *element_data):
+        for each_element in element_data:
+            plus_btn({
+                version.LOWEST: "Add a New Element to this Box",
+                "5.3": "Add a new Element to this Box"})
```

Comment:
We no longer support 5.3 or it's predecessors so it is not required to have a verpick here